src/components/Item.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { useState } from 'react'
+import '../App.css'
+
+export default function Item({ item, onChange, onDelete }) {
+    const[updating, setUpdating] = useState(false)
+
+    let itemInfo
+    if(updating){
+        itemInfo=(
+            <>
+        <input type='text' value={item.text} aria-label='edit-input' name='edit-input'  onChange={(e) => {onChange({ ...item, text: e.target.value, })}
+    }/>
+    <button className='listButton' type='button' onClick={() => setUpdating(false)}>Save Item</button>
```

Comment:
on this onClick should be a editItem function does what the current onChange function is doing and setting the edit mode to false.
something like:

`const editItem(task) {
changeItems(task)
setUpdating(false)
};`